Preprocessing/adcpBinMappingPP.m

```diff
@@ -99,13 +115,13 @@
         %TODO: block function.
         CP = cos(pitch);
         % H[TxB] = P[T] x (+-R[T] x B[B])
-        nonMappedHeightAboveSensorBeam1 = CP .* (cos(beamAngle + roll) / cos(beamAngle) .* distAlongBeams);
-        nonMappedHeightAboveSensorBeam2 = CP .* (cos(beamAngle - roll) / cos(beamAngle) .* distAlongBeams);
+        nonMappedHeightAboveSensorBeam1 = CP .* (cos(beamAngle + sg1*roll) / cos(beamAngle) .* distAlongBeams);
+        nonMappedHeightAboveSensorBeam2 = CP .* (cos(beamAngle - sg1*roll) / cos(beamAngle) .* distAlongBeams);
 
         % H[TxB] = R[T] x (-+P[T] x B[B])
         CR = cos(roll);
-        nonMappedHeightAboveSensorBeam3 = CR .* (cos(beamAngle - pitch) / cos(beamAngle) .* distAlongBeams);
-        nonMappedHeightAboveSensorBeam4 = CR .* (cos(beamAngle + pitch) / cos(beamAngle) .* distAlongBeams);
+        nonMappedHeightAboveSensorBeam3 = CR .* (cos(beamAngle + sg3*pitch) / cos(beamAngle) .* distAlongBeams);
+        nonMappedHeightAboveSensorBeam4 = CR .* (cos(beamAngle - sg3*pitch) / cos(beamAngle) .* distAlongBeams);
```

Comment:
@evacougnon, yes, because when the instrument is inverted, it only affects the pitch (beams 3 and 4), not the roll. Attached two RDI documents which go into detail about how they work. Page 16-17 of the coordinate transformation document explains the beam configurations.

[rdi_primer.pdf](https://github.com/aodn/imos-toolbox/files/7867102/rdi_primer.pdf)
[adcp+coordinate+transformation_Jan10.pdf](https://github.com/aodn/imos-toolbox/files/7867103/adcp%2Bcoordinate%2Btransformation_Jan10.pdf)

.